js/adapt-contrib-text.js

```diff
@@ -1,58 +1,8 @@
-define([
-  'core/js/adapt',
-  'core/js/views/componentView',
-  'core/js/models/componentModel'
-], function(Adapt, ComponentView, ComponentModel) {
+import Adapt from 'core/js/adapt';
+import TextView from './textView';
+import TextModel from './textModel';
```

Comment:
Suggest file renames.

```suggestion
import TextModel from './TextModel';
import TextView from './TextView';
```